### PR TITLE
Add documentation for features which require pre-reviews

### PR DESCRIPTION
**Pull request by renanrodrigo**
As we decided after the (rather traumatic) 27.11 release, some features will need special approvals before we implement them, to avoid problems in the SRU process. This changeset adds that information to the dev docs.

## Desired commit type
 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist
 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -33,3 +33,9 @@ that apply. -->
  - [ ] I have updated or added any unit tests accordingly
  - [ ] I have updated or added any integration tests accordingly
  - [ ] I have updated or added any documentation accordingly
+
+## Does this PR require exra reviews?
```
Comment (basak) [inline .github/PULL_REQUEST_TEMPLATE.md:37]: Typo

**Review by basak — APPROVED** [review]
This looks great. Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (renanrodrigo) [reply .github/PULL_REQUEST_TEMPLATE.md:37]: oops

File: dev-docs/references/early_review_signoff.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Early review sign-off
+As the Client has a very broad exception for SRUs, there are some fetures which
```
Comment (lucasmoura) [inline dev-docs/references/early_review_signoff.md:2]: s/fetures/features

**Review by lucasmoura — APPROVED** [review]

File: dev-docs/howtoguides/how_to_release_a_new_version_of_ua.md [hunk 3]
```diff
@@ -126,26 +129,26 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
...
     h. On Launchpad, create a merge proposal for this version which targets `ubuntu/devel`
-      * For an example, see the [27.9 merge proposal](https://code.launchpad.net/~orndorffgrant/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/422906)
-      * Add 2 review slots for `canonical-server-reporter` and `canonical-server-core-reviewers`
+      * For an example, see the [27.9 merge proposal](https://code.launchpad.net/~orndorffgrant/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/422906).
+      * Add 2 review slots for `canonical-server-reporter` and `canonical-server-core-reviewers`.
 
 4. Server Team Review and Pre-SRU Review
 
-    a. Ask the assigned ubuntu-advantage-tools reviewer/sponsor from Server team for a review of your MPs (If you don't know who that is, ask in ~Server). Include a link to the MP into ubuntu/devel and to the SRU bug.
+    a. Ask the assigned ubuntu-advantage-tools reviewer/sponsor from Server team for a review of your MPs. If you don't know who that is, ask in ~Server. Include a link to the MP and to the SRU bug in ubuntu/devel.
```
Comment (orndorffgrant) [inline dev-docs/howtoguides/how_to_release_a_new_version_of_ua.md:147]: This edit changes the meaning slightly, but the old version is confusing. Here's an alternative suggestion
```suggestion
    a. Ask the assigned ubuntu-advantage-tools reviewer/sponsor from Server team for a review of your MPs. If you don't know who that is, ask in ~Server. Include a link to the ubuntu/devel MP and to the SRU bug.
```

**Review by orndorffgrant — APPROVED** [review]

**Comment by renanrodrigo** [comment]
Plenty approvals, everything addressed, merging

**Pull request merged**